game-core/src/test/java/games/strategy/engine/lobby/client/login/LobbyServerPropertiesFetcherIntegrationTest.java

```diff
@@ -1,21 +0,0 @@
-package games.strategy.engine.lobby.client.login;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.notNullValue;
-
-import org.junit.jupiter.api.Test;
-import org.triplea.test.common.Integration;
-
-import games.strategy.triplea.settings.AbstractClientSettingTestCase;
-
-@Integration
-class LobbyServerPropertiesFetcherIntegrationTest extends AbstractClientSettingTestCase {
-  @Test
-  void remoteLobbyUrlReaderWorks() {
-
-    final LobbyServerPropertiesFetcher testObj = LobbyPropertyFetcherConfiguration.lobbyServerPropertiesFetcher();
-
-    assertThat("verify fetch to get current lobby properties",
-        testObj.fetchLobbyServerProperties(), notNullValue());
-  }
```

Comment:
Could you elaborate on why you removed this test?

Reply:
It seems impossible to make it correct as it's reading a live file. The test fails with:

```
java.lang.NullPointerException: httpsPort is marked @NonNull but is null
```

To fix that, we'd have to modify the live file, to modify the live file, we'd have to merge this PR. Maybe before our validation constraints were not as strong, it was also intended for new properties to be made optional and then eventually required once the file was updated. 

Given the non-null checks and that we force various properties to be present, seems we are in a catch-22 now.